.circleci/config.yml

```diff
@@ -318,3 +335,13 @@ workflows:
               only: /^v([0-9]+)\.([0-9]+)\.([0-9]+)(?:([0-9A-Za-z-]+(?:\.[0-9A-Za-z-]+)*))?(?:\+[0-9A-Za-z-]+)?$/
             branches:
               ignore: /.*/
+
+  announcements:
```

Comment:
maybe we should just put this at the end of the `publish` jobs?  That way the twitter announcement won't get published until all the other jobs successfully publish?  you'll probably want to add a `requires` to this one though..

```
          requires:
            - deploy-dockerhub-tagged
```